website/src/components/Dashboard/WelcomeModal.tsx

```diff
@@ -0,0 +1,64 @@
+import {
```

Comment:
Rather than a Modal can this be a widget that sits on the dashboard? Maybe something below the task Options and above the leaderboard?  Right now with the way the session works the modal would show up every time the user goes to the dashboard during their first login session.  That'd be pretty annoying.  But a widget that quietly sits there wouldn't be so bad.